### ISSUE TITLE
Edition events (insert_at, update_at) doesn't work for polylines

### ISSUE THREAD

**Issue by DzmitryU** [angular-ui/angular-google-maps#1559]
Hello.
I'm using "angular-google-maps 2.1.0-X.8" and "AngularJS v1.3.13" for displaying polylines on map.
I added click, insert_at and update_at events to all polylines, but only click-events works only for editable polylines.
How is it possible to catch event for inserting new point to polyline and relocation of existing points?
Here link to plnkr: http://plnkr.co/edit/gBGI15PxGAhrAVkLu8TR?p=preview

**Comment by nmccready** [comment]
IF your using X.8, that is a beta release. You should be on 2.1.6 or 2.2.1 . The problem is probably the same however. But i am just stating that I have no plans on diving into the older versions to debug them. It basically a just a few of devs and I dont have the time to do that. 

If you absolutely need that older version fixed then you will need to fork the project and dive into the code base.

**Comment by nmccready** [comment]
As for your problem I noticed the plnkr that the undo button is disappearing which is odd. So I am wondering if the entire Polyline get re-created on destroy which probably should not happen.

**Comment by DzmitryU** [comment]
I changed version to 2.1.6, but the result is the same.

**Comment by nmccready** [comment]
Yeah it looks like an issue with at least editing. As for the events I will have to look at your scenario some more.
